Kickstarter-iOS/Views/Cells/SettingsTableViewCell.swift

```diff
@@ -54,4 +50,24 @@ final class SettingsTableViewCell: UITableViewCell, ValueCell, NibLoading {
     _ = self
       |> UITableViewCell.lens.backgroundColor .~ highlightedColor
   }
+
+  private static func descriptionString(for cellType: SettingsCellTypeProtocol) -> String {
```

Comment:
Would it make sense to have these be variables within the `cellType`, similar to how we use `title`? Then it would be called using `cellType.descriptionString`.